src/utils/fetch.ts

```diff
@@ -28,6 +28,7 @@ export const withTimeout = async (
   return res
 }
 
+// TODO: put in env vars for staging
```

Comment:
@sddioulde do you think we can drop this now?

Reply:
@sddioulde feel free to change in another PR 🙃